droneModels.json

```diff
@@ -317,39 +315,39 @@
       "heightPixels": 3032,
       "comment": "DJI Mavic 2 Enterprise Dual",
       "lensType": "perspective",
-      "radialR1": 0,
-      "radialR2": 0,
-      "radialR3": 0,
-      "tangentialT1": 0,
-      "tangentialT2": 0
+      "radialR1": 0.181996,
+      "radialR2": -0.593144,
+      "radialR3": 0.418029,
+      "tangentialT1": 0.00112363,
+      "tangentialT2": -0.000343339
     },
     {
       "makeModel": "djiFC2403",
       "ccdWidthMMPerPixel": "6.49952/4056.0",
       "ccdHeightMMPerPixel": "4.944222/3040.0",
       "widthPixels": 4056,
       "heightPixels": 3040,
-      "comment": "DJI Mavic 2 Enterprise Dual thermal sensor, Teledyne FLIR Lepton 3.5 sensor, 160x120 px sensor, upscaled by software to 640x480 px",
+      "comment": "DJI Mavic 2 Enterprise Dual (color camera)",
       "lensType": "perspective",
-      "radialR1": 0,
-      "radialR2": 0,
-      "radialR3": 0,
-      "tangentialT1": 0,
-      "tangentialT2": 0
+      "radialR1": -0.02148245,
+      "radialR2": -0.000571985,
+      "radialR3": 0.015372923,
+      "tangentialT1": 0.000306293,
+      "tangentialT2": -0.00146070
     },
     {
       "makeModel": "djiFC3170",
-      "ccdWidthMMPerPixel": "6.623518/8000.0",
-      "ccdHeightMMPerPixel": "4.854492/6000.0",
-      "widthPixels": 8000,
-      "heightPixels": 6000,
+      "ccdWidthMMPerPixel": "6.623518/4000.0",
```

Comment:
Will need to revisit this later. There are some pretty specific formulas for how we need to scale R and T distortion parameters, it is non-linear with the image scale